modules/cluster-logging-uninstall.adoc

```diff
@@ -60,9 +60,33 @@ Do not delete the `openshift-operators-redhat` project if other global operators
 
 .. Confirm the deletion by typing `openshift-operators-redhat` in the dialog box and click *Delete*.
 
-. Optional: Remove any OpenShift Logging persistent volume claims (PVC):
+.Next steps
+
+Uninstalling OpenShift Logging does not remove the persistent volume claims (PVCs). To preserve or delete the PVCs, persistent volumes (PVs), and associated data, you must take further action.
+
+
+[WARNING]
+====
+Releasing or deleting PVCs can delete PVs and cause data loss.
+====
+
+. Evaluate whether to keep the PVCs for reuse with other pods. If so, keep the labels or PVC names you need to reclaim the PVCs.
+
+. Optional: If you do not want to reclaim the PVCs, you can delete them.
++
+If a PV's `persistentVolumeReclaimPolicy` is `delete`, deleting the PVC deletes its PV. This action releases the backing storage and removes all data, such as collected log data.
 
 .. Switch to the *Storage* -> *Persistent Volume Claims* page.
 
 .. Click the Options menu {kebab} next to each PVC and select *Delete Persistent Volume Claim*.
 
+.. If the PV's `persistentVolumeReclaimPolicy` is `retain`, use the `kubectl delete pv` command to delete the PVs. For example, enter:
++
+[source,terminal]
+----
+$ kubectl delete pv --all
+----
+
+.Additional resources
```

Comment:
My understanding is that the link provided in this section should be a cross-reference, not a link. This is because the target URL is part of the OpenShift docs library. You could move this **Additional resources** section and its contents to the `logging/cluster-logging-uninstall.adoc` assembly, directly under the `include` line that points to this module. This will enable the use of an `xref` instead of a link.

The `xref` in `logging/cluster-logging-uninstall.adoc` I think would be as follows, although I haven't tested it:

`* xref:../storage/understanding-persistent-storage.adoc#reclaim-policy_understanding-persistent-storage[Changing the reclaim policy of a persistent volume]`

Given that there is only one module in the assembly, you could use `== Additional resources` (in the assembly file) instead of `.Additional resources` and provide a unique label for that title, such as:

`[id="cluster-logging-uninstall-additional-resources"]`

Reply:
@rolfedh another reason why this should be a cross-reference is that the link will take the reader to the latest version of the target section. A cross-reference will stay within the same release. This content is labelled to be cherry picked into multiple enterprise releases.

Reply:
On principle, a module should be self-contained and portable. The link relates directly to the content of the module and, otherwise, does not relate to that of the assembly. For example, if we were to move the module or include another module below it, the link would be misplaced. Also, the Hn+1 _looks_ out of place, putting too much separation between the module content and the related link. To stay within the release, I've replaced `4.7` and then `latest` with {product-version}.

Reply:
@rolfedh if an Hn+1 title looks out of place, you could use a block title in the assembly for the `.Additional resources` potentially.

Also, if you want to add another module in the future, you could add it underneath the additional resource content in the assembly. I have done something [in this assembly](https://docs.openshift.com/container-platform/4.7/monitoring/configuring-the-monitoring-stack.html) several times, where the additional resources links relate to a specific module.

Reply:
Hi @rolfedh, @pneedle-rh is right. 

This is circumventing the ban on xrefs within modules by using a link tag in a creative way, and even though I continue to be on the fence about the use of xrefs within modules, the net effect of this is to add a link at a location where we are currently just not allowed to. As soon as the mod docs team allows xrefs within modules, **and** the PV2 infrastructure has a validation mechanism in place for valid xrefs, we can revisit this. Happy to jump on call to discuss.

For now, you will need to remove this link.

Reply:
Resolved per our conversation.